reference-architecture/gce-cli/ansible/playbooks/roles/rhel-image/defaults/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+rhel_image_path: '{{ ansible_env.HOME }}/Downloads/rhel-guest-image-7.3-35.x86_64.qcow2'
```

Comment:
Also I should note, that later this variable will be probably moved to main config file, similar as currently `config.sh.example`, so it will have better visibility.